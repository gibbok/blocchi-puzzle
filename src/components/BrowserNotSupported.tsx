import React from 'react';
import styled from 'styled-components';

const BrowserNotSupportedStyled = styled.div`
  width: 100vw;
  height: 100vh;
  font-family: 'Oleo Script', cursive;
  color: #442217;
  display: flex;
  justify-content: center;
  align-items: center;
  text-align: center;
  flex-direction: column;
`;

const Main = styled.div`
  font-size: 5rem;
`;

const Sub = styled.div`
  font-size: 3rem;
`;

export function BrowserNotSupported(): JSX.Element {
  return (
    <BrowserNotSupportedStyled>
      <Main>Browser not supported! ðŸ˜±</Main>
      <Sub>Please use the latest Chrome or Firefox.</Sub>
    </BrowserNotSupportedStyled>
  );
}
