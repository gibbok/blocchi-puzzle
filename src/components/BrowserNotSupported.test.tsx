import renderer from 'react-test-renderer';
import { BrowserNotSupported } from './BrowserNotSupported';
import React from 'react';

describe('<BrowserNotSupported />', () => {
  it('should render', () => {
    const tree = renderer.create(<BrowserNotSupported />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
