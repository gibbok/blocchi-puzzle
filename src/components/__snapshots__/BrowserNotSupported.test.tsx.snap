// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BrowserNotSupported /> should render 1`] = `
<div
  className="sc-AxjAm gHDWZW"
>
  <div
    className="sc-AxirZ hbTpnX"
  >
    Browser not supported! ðŸ˜±
  </div>
  <div
    className="sc-AxiKw cFEFMp"
  >
    Please use the latest Chrome or Firefox.
  </div>
</div>
`;
